Userland/Libraries/LibWeb/Geometry/DOMMatrixReadOnly.h

```diff
@@ -56,6 +56,8 @@ class DOMMatrixReadOnly : public Bindings::PlatformObject {
     virtual ~DOMMatrixReadOnly() override;
 
     static WebIDL::ExceptionOr<JS::NonnullGCPtr<DOMMatrixReadOnly>> from_matrix(JS::VM&, DOMMatrixInit& other);
+    static WebIDL::ExceptionOr<JS::NonnullGCPtr<DOMMatrixReadOnly>> from_float32_array(JS::VM&, JS::Handle<JS::Object> const& array32);
+    static WebIDL::ExceptionOr<JS::NonnullGCPtr<DOMMatrixReadOnly>> from_float64_array(JS::VM&, JS::Handle<JS::Object> const& array64);
```

Comment:
Seems fine, one suggestion: in the future (and especially if there's more APIs to add) it might make sense to create some template functions for these that do `from_float64_array<DOMMatrixReadOnly>(...)` or `from_float64_array<DOMMatrix>(...)` to reduce the duplication.